js/get-data.js

```diff
@@ -0,0 +1,11 @@
+const urlData = 'https://25.javascript.pages.academy/keksobooking/data';
+async function getData (url) {
+  let obj = [];
+  await fetch(url).then((responce) => responce.json())
+    .then((data) => {
+      obj = data;
+    });
```

Comment:
кажется, что тут можно проще 
```suggestion
  const fetchedOffer = await fetch(url)
  return await fetchedOffer.json();
```